sdk/search/search/src/searchIndexClient.ts

```diff
@@ -480,15 +477,38 @@ export class SearchIndexClient<T> {
     documents: T[],
     options: MergeDocumentsOptions = {}
   ): Promise<IndexDocumentsResult> {
-    const actionType = options.uploadIfNotExists ? "mergeOrUpload" : "merge";
     const { span, updatedOptions } = createSpan("SearchIndexClient-mergeDocuments", options);
 
-    const batch = documents.map<IndexAction<T>>((doc) => {
-      return {
-        ...doc,
-        actionType
-      };
-    });
+    const batch = new IndexDocumentsBatch<T>();
+    batch.merge(documents);
+
+    try {
+      return await this.indexDocuments(batch, updatedOptions);
+    } catch (e) {
+      span.setStatus({
+        code: CanonicalCode.UNKNOWN,
```

Comment:
Should we be aligning on this across languages for consistent telemetry?  I think we're probably sending up `null` or an empty string in .NET right now.

Reply:
The default error code is `UNKNOWN` in the OT spec I believe. @pakrym can confirm

Reply:
We use Unknown:

https://github.com/open-telemetry/opentelemetry-dotnet/blob/5f12087e07f298824f6484ec890aa25db0b83a4d/src/OpenTelemetry.Collector.Dependencies/Implementation/AzureSdkDiagnosticListener.cs#L121

Reply:
Oh, got it.  I thought this was related to the Error.Code property which Search currently leaves null.  Apologies for the random noise.